source/Magritte-Deprecated3dot7/MABlockDescription.extension.st

```diff
@@ -0,0 +1,6 @@
+Extension { #name : #MABlockDescription }
+
+{ #category : #'*Magritte-Deprecated3dot7' }
```

Comment:
IIUC this method belongs in the Morphic package

Reply:
I think I put it to this package, because it was referenced only by this package. Maybe I remember it incorrectly.